WooCommerce/WooCommerceTests/ViewRelated/ReusableViews/TextFieldTextAlignmentTests.swift

```diff
@@ -5,7 +5,7 @@ final class TextFieldTextAlignmentTests: XCTestCase {
 
     // MARK: .leading
 
-    func testTextFieldTextLeadingAlignmentMapsToLeftForLTR() {
+    func test_TextFieldText_leading_alignment_maps_to_left_for_LTR() {
```

Comment:
nit: maybe `test_TextField_text_leading_alignment_maps_to_left_for_LTR` since `TextField` refers to a text field? (similar for other tests in this file)

Reply:
Thanks. I was not sure about how to do this. 😄 Fixed in c464599b9c98112a7a239f74488b19f248e1750c.